src/models/imposterPrinter.js

```diff
@@ -30,6 +30,9 @@ function create (header, server, loadRequests) {
         if (header.endOfRequestResolver) {
             result.endOfRequestResolver = header.endOfRequestResolver;
         }
+        if (header.allowCORS) {
+            result.allowCORS = header.allowCORS;
+        }
```

Comment:
I doubted a little bit if this was the right place to put this, since its http(s) only, but I guess so since since tcp's `endOfRequestResolver` is also listed here on line 30?